processor/resourcedetectionprocessor/README.md

```diff
@@ -40,6 +40,19 @@ to read resource information from the [GCE metadata server](https://cloud.google
     * host.name
     * host.type
 
+It also can optionally gather tags for the EC2 instance that the collector is running on
+
+EC2 custom configuration example:
+```yaml
+detectors: ["ec2"]
+ec2:
+    # A list of regex's to match tags to add as resource attributes can be specified
+    tags:
+        - tag1
+        - tag2
```

Comment:
What's the intended meaning of adding `tag1` to the list? With the current implementation, this would for example match a tag with key `tag12`([see other examples here](https://play.golang.org/p/n14gXlu_gre)), and that might be confusing for the end-user.

I would either
1. Explicitly add `^tag1$` as an example to show that to have an exact match you need to add it in the regex or
2. Switch to something like what the filterprocessor does (see [here](https://github.com/open-telemetry/opentelemetry-collector/blob/master/processor/README.md#includeexclude-metrics)), having different strict and regex match types.

Reply:
Good catch, thanks, I will do option 1 for now and open up an issue for option 2 since it requires more changes outside of this repo and will make this PR more complex